### ISSUE TITLE
Triage and fix `noqa` directives

### ISSUE THREAD

**Issue by lzaoral** [openscanhub/openscanhub#104]
This project now contains the following noqa directives obtained using `git grep -i 'noqa'`. This issue serves as a tracking bug for all of them (triaged/fixed instances are marked using ✅):

- [x] `osh/client/commands/cmd_diff_build.py:    def run(self, *args, **kwargs):  # noqa: C901`
  * https://github.com/openscanhub/openscanhub/pull/50
- [x] `osh/client/commands/cmd_version_diff_build.py:    def run(self, *args, **kwargs):  # noqa: C901`
  * https://github.com/openscanhub/openscanhub/pull/50
- [x] `osh/hub/errata/check.py:    except:  # noqa: B901, E722`
  * https://github.com/openscanhub/openscanhub/pull/139
- [ ] `osh/hub/errata/scanner.py:    except Exception as ex:  # noqa: B902`
- [ ] `osh/hub/osh_xmlrpc/errata.py:    except Exception as ex:  # noqa: B902`
- [x] `osh/hub/other/autoregister.py:def autoregister_admin(module, exclude_models=None, model_fields=None, exclude_fields=None,  # noqa: C901`
    * https://github.com/openscanhub/openscanhub/issues/132
- [ ] `osh/hub/scan/admin.py:        except Exception as e: # noqa`
- [ ] `osh/hub/scan/xmlrpc_helper.py:    except Exception as ex:  # noqa: B902`
- [ ] `osh/hub/settings.py:    from .settings_local import *  # noqa`
    * We only want to ignore the star import here and not completely everything.
- [ ] `osh/hub/waiving/results_loader.py:            except:  # noqa: B901, E722`

**Comment by kdudka** [comment]
I do not think we need to fix these issues proactively (and the above list might already be outdated).
